package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^3.0.0"
```

Comment:
Does it require a webpack major version bump? If `.getChunks` is available in 2.x, we could do the change as a minor GWP update, otherwise we'll need a major bump, therefore we could keep track of these changes in a GWP 2.x developing branch for now.